subprojects/composite-builds/src/integTest/groovy/org/gradle/integtests/composite/CompositeContinuousBuildIntegrationTest.groovy

```diff
@@ -61,7 +61,7 @@ class CompositeContinuousBuildIntegrationTest extends AbstractContinuousIntegrat
         when:
         inputFile.text = "second"
         then:
-        succeeds()
+        successfulBuildTriggered()
```

Comment:
Should we have `buildTriggeredAndIsSuccessful()` instead?

Reply:
I changed it to `buildTriggeredAndSucceeded()`. For a failing build it is now `buildTriggeredAndFailed()`.

I think the name for a failing build is good now. For successful builds, I considered:
- `buildTriggeredAndIsSuccessful()`
- `buildTriggeredAndWasSuccessful()`
- `buildTriggeredAndFinishedSuccessful()`
- `buildTriggeredAndSucceeded()`

I ended up choosing the last one. WDYT?